### ISSUE TITLE
UI: Show Person Attendances in tabular format

### ISSUE THREAD

**Issue by chrisgzf** [AY2021S1-CS2103T-F11-1/tp#72]

**Comment by chrisgzf** [comment]
This supersedes #7, closing #7 and reassigning this to @rnmy 

tldr: in #73, attendance data field is added to Person and saved with Person, when serialised into the json file. right now, you need to implement the:
1. UI which shows student attendances - for this, its up to you, but i have an idea where basically we use tabbed navigation, the main tab is the student's details, and the second tab is attendance. inside the attendance tab, we will see a table of all the students (their names in the first column), and the attendance dates are in the row

Something like:
| Name | 15/10 | 22/10|
| - | - | - |
| Chris | 🚫 | ✅ |
| Rach | ✅ | ✅ |

2. maybe the `showatt` command which navigates to that tab (the UG `showatt` command currently is for showing a specific date's attendance, maybe we can change that to simply navigating to the attendance page/tab instead, idk)
